tests/test_appsettings.py

```diff
@@ -19,21 +22,30 @@ class SettingTestCase(SimpleTestCase):
     def _imported_object2():
         return "nothing"
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        fd, cls.temporary_file_name = mkstemp()
+
+    @classmethod
+    def tearDownClass(cls):
+        os.remove(cls.temporary_file_name)
+
     def setUp(self):
         self.message_required = "%s setting is required and"
         self.message_missing_item = "%s setting is missing required item"
         self.message_no_attr = "has no attribute '%s'"
 
     def test_setting(self):
-        setting = appsettings.Setting(name='simple')
-        assert setting.name == 'simple'
-        assert setting.full_name == 'SIMPLE'
+        setting = appsettings.Setting(name="simple")
+        assert setting.name == "simple"
+        assert setting.full_name == "SIMPLE"
```

Comment:
This seems to be an unrelated change that has nothing to do with `FileSetting`.

Reply:
There are more of those.

Reply:
Those are changes from `black` and `flake8` because of `tox`. Should I remove the changes and let the `tox` fail?

Reply:
It doesn't seem to be failing on master.

Reply:
It's because CI on GitLab was temporarily disabled.

Reply:
I was a bit behind `pawamoy/master`, pulled and rebased. But there were still 3 changes in tests, which had to be fixed.

Reply:
I'd leave that for some cleaning commit/PR.

Reply:
Another pull seems to have fixed it.